### PR TITLE
javascript-forms | Samuel Cho

### PR DESCRIPTION
**Pull request by SamuelCho1**
Used JavaScript to gather user input from HTML forms.
Screen Shot 2021-02-03 at 11 49 55 AM
Screen Shot 2021-02-03 at 11 50 09 AM
Screen Shot 2021-02-03 at 11 50 26 AM
Screen Shot 2021-02-03 at 12 27 13 PM

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var $contactForm = document.querySelector('#contact-form');
+var messageData = {};
```
Comment (codingcodymiller) [inline javascript-forms/form-values.js:2]: This should be inside the function block.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! One minor note:

**Comment by SamuelCho1** [comment]
Screen Shot 2021-02-03 at 12 46 13 PM

**Review by codingcodymiller — APPROVED** [review]
Nicely done!

**Pull request merged**